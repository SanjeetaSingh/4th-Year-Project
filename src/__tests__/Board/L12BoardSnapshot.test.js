import renderer from 'react-test-renderer'

import Level12 from '../../Level12/L12-Board/L12-Board'

it('snapshot test level 12', () => {
    const tree =  renderer.create(<Level12 />).toJSON()
    expect(tree).toMatchSnapshot();
});