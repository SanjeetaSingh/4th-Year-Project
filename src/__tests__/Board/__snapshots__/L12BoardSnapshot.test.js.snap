// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test level 12 1`] = `
<div
  className="container"
>
  <div
    className="boardcontainer2"
  >
    <div
      className="board11"
    >
      <div
        className="box"
        data-col={0}
        data-row={0}
      />
      <div
        className="box"
        data-col={1}
        data-row={0}
      >
        <img
          alt="f"
          class="bone"
          id="food"
          src="assets/food.png"
        />
      </div>
      <div
        className="box"
        data-col={2}
        data-row={0}
      />
      <div
        className="box"
        data-col={0}
        data-row={1}
      />
      <div
        className="box"
        data-col={1}
        data-row={1}
      />
      <div
        className="box"
        data-col={2}
        data-row={1}
      />
      <div
        className="box"
        data-col={0}
        data-row={2}
      />
      <div
        className="box"
        data-col={1}
        data-row={2}
      >
        <img
          alt="d"
          class="image"
          id="dog"
          src="assets/dog.png"
        />
      </div>
      <div
        className="box"
        data-col={2}
        data-row={2}
      />
    </div>
  </div>
  <div
    className="bottom-panel"
  >
    <div
      class="border"
    >
      <div
        class="level5Contain"
      >
        <h2>
          Level 12:
        </h2>
        <div
          class="speech"
        >
          This level introduces variables! For the values of the varibles you need to set coordinates for the row and column of the bowl to help the dog get to the food. These varibales will be used in the while loop as a condtion to stop looping through the commands. You have to use the same amount or less amount of commands mentioned in the top right corner.
          <br />
           Good luck! 
          <br />
           (To learn more about while and to understand the while loop condition and the how variables work click the i below)
        </div>
        <div
          class="borderPanel"
        >
          <div
            class="toppart12"
          >
            <div
              class="containing"
            >
              <p
                class="number"
                data-testid="counter"
                id="count"
              >
                0/2
              </p>
            </div>
            <p
              class="titles"
            >
              Enter Sequence:
            </p>
            <br />
            <div
              class="buttons-wrapper12"
            >
              <button
                class="seemingly-inner-button12"
                disabled={false}
                onClick={[Function]}
              >
                <i
                  class="fa fa-info"
                />
              </button>
              <p
                class="subtitles"
              >
                Variable Declaration
              </p>
            </div>
            <form
              class="var"
            >
              int row =  
              <input
                class="input"
                id="var1"
                onChange={[Function]}
                size="5"
              />
                 ; 
              <br />
              <br />
              int col = 
              <input
                class="input"
                id="var2"
                onChange={[Function]}
                size="5"
              />
                ;
              <br />
              <br />
            </form>
            <p
              class="lists"
              id="action"
            />
          </div>
        </div>
        <div
          class="outside"
        >
          <button
            class="button"
            disabled={false}
            onClick={[Function]}
            type="submit"
          >
            Left
          </button>
          <button
            class="button"
            disabled={false}
            onClick={[Function]}
            type="submit"
          >
            Up
          </button>
          <button
            class="button"
            disabled={false}
            onClick={[Function]}
            type="submit"
          >
            Down
          </button>
          <button
            class="button"
            disabled={false}
            onClick={[Function]}
            type="submit"
          >
            Right
          </button>
          <div
            class="buttons-wrapper5"
          >
            <button
              class="seemingly-inner-button"
              disabled={false}
              onClick={[Function]}
            >
              <i
                class="fa fa-info"
              />
            </button>
            <button
              class="button"
              disabled={false}
              onClick={[Function]}
              type="submit"
            >
              While
            </button>
          </div>
          <button
            class="button"
            disabled={false}
            onClick={[Function]}
            type="submit"
          >
            Clear
          </button>
          <button
            class="button"
            disabled={false}
            onClick={[Function]}
            type="submit"
          >
             Submit
          </button>
        </div>
      </div>
    </div>
  </div>
</div>
`;
